mixer/template/tracespan/tracespan.proto

```diff
@@ -115,4 +115,54 @@ message Template {
     // not provided for server spans.
     // Optional
     bool client_span = 9;
+
+    // Identifies the source (client) of this span.
+    // Should usually be set to `source.workload.name`.
+    //
+    // Optional.
+    string source_name = 10;
+
+    // Client IP address. Should usually be set to `source.ip`.
+    //
+    // Optional.
+    istio.policy.v1beta1.IPAddress source_ip = 11;
+
+    // Identifies the destination (service) of this span.
```

Comment:
nit: s/(service)/(workload)

Reply:
meant to be "server", not "service" (as in "server side")